packages/vscode-extension/messages.json

```diff
@@ -46,6 +49,7 @@
 
 	"DISMISS": "Dismiss",
 
+	"ERRORS_COURRUPTED_SDK_JAR_DEPENDENCY": "There was a problem with SuiteCloud Extension dependencies. Restart your Visual Studio Code instance.",
```

Comment:
Was this reviewed by Montsant?

Reply:
Yes it was reviewed by Montsant. The capitalized words on the buttons section, was also reviewed by her.